File: MainInclude/mainExo5.java

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int result = countDigits(n);
        System.out.println("Le nombre de chiffres de " + n + " est " + result);
        scanner.close();
    }
}

File: MainInclude/mainExo5.js
function main() {
    const rl = readline.createInterface({
        input: process.stdin,
        output: process.stdout
    });

    rl.question("", (input) => {
        const number = parseInt(input, 10);
            const result = countDigits(number);
            console.log(`Le nombre de chiffres de ${number} est ${result}`);
        rl.close();
    });
}
main();
